lib/decidim/extra_user_fields/engine.rb

```diff
@@ -11,20 +11,25 @@ module ExtraUserFields
     class Engine < ::Rails::Engine
       isolate_namespace Decidim::ExtraUserFields
 
+      require "extends/presenters/decidim/admin_log_organization_presenter_extends"
```

Comment:
We should require extends in [extra_user_fields.rb](https://github.com/AyakorK/decidim-module-extra_user_fields/blob/master/lib/decidim/extra_user_fields.rb) file